src/ngraph/runtime/gpu/gpu_call_frame.cpp

```diff
@@ -32,9 +32,6 @@ runtime::gpu::GPU_CallFrame::GPU_CallFrame(std::shared_ptr<GPU_ExternalFunction>
     : m_external_function(external_function)
     , m_compiled_function(compiled_function)
 {
-    //Create context use driver API and make it current, the runtime call will pickup the context
-    //http://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#interoperability-between-runtime-and-driver-apis
-    ngraph::runtime::gpu::CudaContextManager::instance();
     cublasStatus_t cublasStatus = cublasCreate(&m_cublas_handle);
```

Comment:
we can also move `cublasCreate` and `cudnnCreate` to `gpu_runtime_context.hpp`, so we don't need the `link_runtime_context`;